### ISSUE TITLE
elasticsearch 2.3.5 : can't install elasticsearch-head as a plugin

### ISSUE THREAD

**Issue by etienne-sf** [mobz/elasticsearch-head#267]
Hello,

  I just tried to install elasticsearch-head as a plugin.

I'm using elasticsearch 2.3.5, on Windows, with JDK8.

I went in the bin elasticsearch folder (for me : H:\java\elasticsearch\bin), and executed the following command, as explained on the elasticsearch-head doc :
plugin -install H:\data\git\elasticsearch-head
but I got this error: 
ERROR: unknown command [-install]. Use [-h] option to list available commands
(typo in the doc ?)

So I tried:
plugin install H:\data\git\elasticsearch-head

But I got this output:
-> Installing H:\data\git\elasticsearch-head...
ERROR: Invalid prefix or suffix

Following elasticsearch help, I tried:
plugin install file:/H:/data/git/elasticsearch-head

But I got:
-> Installing from file:/H:/data/git/elasticsearch-head...
Trying file:/H:/data/git/elasticsearch-head ...
Downloading .DONE
Verifying file:/H:/data/git/elasticsearch-head checksums if available ...
NOTE: Unable to verify checksum for downloaded plugin (unable to find .sha1 or .md5 file to verify)
ERROR: Could not find plugin descriptor 'plugin-descriptor.properties' in plugin zip

Seems like installing the plugin doesn't work anymore...

Etienne

**Comment by philipskokoh** [comment]
You can ignore that 'plugin-descriptor.properties' error. Please try to open elasticsearch-head in your browser. Are you able to open it?

If elasticsearch-head is not installed properly, can you try installing using this:
`plugin install mobz/elasticsearch-head`

**Comment by etienne-sf** [comment]
Hi,

  Thanks for your answer.

I let my brain one night on it :
- Apparently, installing a plugin must be from a zip. And cloning git doesn't provide a zip
- I then retry from your sample, but with the proper / and \. As I'm under windows, I must have anti-slash in the local path ... not in the plugin name. The following command just works for me:
elasticsearch\bin\plugin install mobz/elasticsearch-head 

So, now : IT's OK.


(In my previous try, I tried to replace mobz/elasticsearch-head  by a local path, whithout success)


Then : 
Accessing directly from the browser didn't work. I could not manage to let it connect to the cluster.

I don't know why. I added the next lines (taken from the elasticsearch doc) in the elasticsearch.yml config file
http.cors.enabled: true
http.cors.allow-origin: /http?:\/\/localhost(:[0-9]+)?/

But I get : nothing setting the elasticsearch URL in head to http://localhost:9200/, and an exception in elasticsearch logs when setting the elasticsearch URL in head to https://localhost:9200/
